test/suite/test_hs14.py

```diff
@@ -75,6 +75,9 @@ def test_hs14(self):
             cursor[self.create_key(i)] = value4
             self.session.commit_transaction('commit_timestamp=' + timestamp_str(4))
 
+        # A checkpoint will esnure older values are writting to the history store.
```

Comment:
Typo: `esnure => ensure` and `writting => writing`

Reply:
Good catch - not sure though why our s_all didn't catch it unless these are some valid dictionary words :). Will fix it